go-controller/pkg/ovn/master.go

```diff
@@ -880,10 +877,12 @@ func (oc *Controller) syncNodesPeriodic() {
 
 	for nodeName, chassisName := range chassisMap {
 		if chassisName != "" {
-			_, stderr, err = util.RunOVNSbctl("--if-exist", "chassis-del", chassisName)
+			cmd, err := oc.ovnSBClient.ChassisDel(chassisName)
 			if err != nil {
-				klog.Errorf("Failed to delete chassis with name %s for node %s: stderr: %s, error: %v",
-					chassisName, nodeName, stderr, err)
+				klog.Errorf("Unable to create the ChassisDel command for chassis: %s from the sbdb", chassisName)
+			} else if err = oc.ovnSBClient.Execute(cmd); err != nil {
```

Comment:
consider: you can also call Execute outside the for-loop. That way we only make a single transaction out to ovsdb.